mapper-runtime/src/main/java/com/datastax/oss/driver/api/mapper/RuntimeAttributes.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.api.mapper;
+
+import com.datastax.oss.driver.api.core.ConsistencyLevel;
+import com.datastax.oss.driver.api.core.config.DriverExecutionProfile;
+import com.datastax.oss.driver.api.core.metadata.Node;
+import java.nio.ByteBuffer;
+import java.time.Duration;
+import java.util.Map;
+
+public interface RuntimeAttributes {
+
+  boolean isExecutionProfileSet();
```

Comment:
Can't we use the same semantics as `Statement/Request`?
E.g. `executionProfile == null` means unset, `pageSize <= 0` means use the default, etc.

Reply:
I had two reasons I did this.
1. I didn't want to duplicate that logic here. Putting in all the defaults here means that if they ever change we are going to have to change them here as well. I wanted to avoid that. With this solution the defaults can change, and this should continue to function appropriately.
2. Boolean state is more complicated, because we can't really infer anything, so we need a separate value to track whether or not they intended to set anything.

Reply:
The defaults are very unlikely to change on `Statement` at this point; while technically not a breaking change it would be a serious incompatibility with previous versions. If anything, I find it more intuitive to have something that behaves exactly like the core statement.

For boolean values, `Statement` uses `Boolean` for idempotence, with null meaning "use the config", and `boolean` for tracing because the default is false.